frontend/src/container/IngestionSettings/MultiIngestionSettings.tsx

```diff
@@ -2,6 +2,7 @@
 /* eslint-disable jsx-a11y/click-events-have-key-events */
```

Comment:
Can we remove this rule?
This works only for html elements right now. Our custom components like Button are not being enforced by this rule